src/main/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/dto/LocationDto.kt

```diff
@@ -109,6 +109,12 @@ data class Location(
   @Schema(description = "Top Level Location Id", example = "57718979-573c-433a-9e51-2d83f887c11c", required = true)
   val topLevelId: UUID,
 
+  @Schema(description = "Current Level with hierarchy", example = "1", required = true)
```

Comment:
Worth noting it's zero-indexed?

Reply:
It starts at 1. Wing = 1, Landing =2, Cell=3

Reply:
But yes could explain more